tidalapi/request.py

```diff
@@ -151,26 +151,13 @@ def request(
         log.debug("request: %s", request.request.url)
         try:
             request.raise_for_status()
-        except Exception as e:
+        except requests.HTTPError as e:
             log.info("Request resulted in exception {}".format(e))
             self.latest_err_response = request
-            if request.content:
-                resp = request.json()
-                # Make sure request response contains the detailed error message
-                if "errors" in resp:
-                    log.debug("Request response: '%s'", resp["errors"][0]["detail"])
-                elif "userMessage" in resp:
-                    log.debug("Request response: '%s'", resp["userMessage"])
-                else:
-                    log.debug("Request response: '%s'", json.dumps(resp))
-
-            if request.status_code and request.status_code == 404:
-                raise ObjectNotFound
-            elif request.status_code and request.status_code == 429:
-                raise TooManyRequests
```

Comment:
From my quick glance at the code, I assume ObjectNotFound and TooManyRequests will still be thrown correctly, through from_http_error?

Reply:
They are not throw just converted and than reraised. Renamed the function to `http_error_to_tidal_error` to make this more clear.

Reply:
Yeah, reraised is what I mean. As long as it is still the case, it should be OK 👍